open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/AssetContextHandler.java

```diff
@@ -87,6 +87,24 @@ public AssetContextHandler(InvalidParameterHandler invalidParameterHandler,
         return repositoryHandler.getEntitiesByType(userId, typeDefGUID, 0, 0, methodName);
     }
 
+    /**
+     * @param userId                 the user id
+     * @param entityTypeName         the name of the entity type
+     * @param findEntitiesParameters filtering used to reduce the scope of the search
+     * @return a list with entities matching the supplied parameteres;
+     * @throws UserNotAuthorizedException the user is not authorized to make this request.
+     * @throws PropertyServerException    something went wrong with the REST call stack.
+     */
+    public List<EntityDetail> getEntitiesToUpdate(String userId, String entityTypeName, FindEntitiesParameters findEntitiesParameters)
```

Comment:
Can we move this into HandlerHelper, or create a more generic handler? AssetContextHandler is used for building the asset context for schema elements, this meths will be used for other entityTypeNames(GlossaryTerm, Process) and it might be misleading to have it here

Reply:
Moved the method logic to handlerHelper and wrapped it here